docs/src/index.rst

```diff
@@ -46,6 +46,8 @@ If you have a specific question, `post a note on the discussion board <https://d
    guides/workflow-config-file
    guides/customizing-visualization
    guides/run-analysis-on-terra
+   guides/run-genbank-ingest-on-terra   
+   guides/run-gisaid-ingest-on-terra  
```

Comment:
Suggestion: have a parent page **Ingest SARS-CoV-2 Data on Terra** which contains the intro text that is currently duplicated. You can then link to the sub-pages using a `.. toctree::` on that page, and the hierarchy should also be reflected in the main sidebar.

URLs might look like:

- `projects/ncov/en/latest/guides/ingest-data-on-terra`
- `projects/ncov/en/latest/guides/ingest-data-on-terra/genbank`
- `projects/ncov/en/latest/guides/ingest-data-on-terra/gisaid`